Tribler/Core/Modules/restapi/create_torrent_endpoint.py

```diff
@@ -0,0 +1,97 @@
+import ast
+import json
+import base64
+import logging
+
+from twisted.web import http, resource
+from twisted.web.server import NOT_DONE_YET
+
+from Tribler.Core.Modules.restapi.util import return_handled_exception
+
+
+class CreateTorrentEndpoint(resource.Resource):
+    """
+    Create a torrent file from local files.
+    See: http://www.bittorrent.org/beps/bep_0012.html
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+        self._logger = logging.getLogger(self.__class__.__name__)
+        self.session = session
+
+    def render_GET(self, request):
+        """
+            .. http:get:: /createtorrent
+
+            Create a torrent from local files and return it in base64 encoding.
+            This endpoint returns a 500 HTTP response if a source file does not exist.
+
+                **Example request**:
+
+                .. sourcecode:: none
+
+                    curl -X PUT http://localhost:8085/mychannel/createtorrent
+                            --data "files=['path/to/file.txt', 'path/to/another/file.mp4']
```

Comment:
I guess not, but why not just showing what the JSON should look like instead of a specific command line tool?